dataset_builders/pie/cdcp/cdcp.py

```diff
@@ -0,0 +1,159 @@
+import dataclasses
+import logging
+from typing import Any, Callable, Dict, List, Optional
+
+import datasets
+import pytorch_ie.data.builder
+from pytorch_ie.annotations import BinaryRelation, LabeledSpan
+from pytorch_ie.core import Annotation, AnnotationList, Document, annotation_field
+from pytorch_ie.documents import TextDocumentWithLabeledSpansAndBinaryRelations
+
+from pie_datasets.document.processing.text_span_trimmer import trim_text_spans
+
+log = logging.getLogger(__name__)
+
+
+def dl2ld(dict_of_lists):
+    return [dict(zip(dict_of_lists, t)) for t in zip(*dict_of_lists.values())]
+
+
+def ld2dl(list_of_dicts, keys: Optional[List[str]] = None, as_list: bool = False):
+    if keys is None:
+        keys = list_of_dicts[0].keys()
+    if as_list:
+        return [[d[k] for d in list_of_dicts] for k in keys]

```

Comment:
remove this option, this is used nowhere

Reply:
fix in 0a66d06